packages/cxl-lumo-styles/scss/themes/vaadin-tabs.scss

```diff
@@ -65,3 +65,43 @@ $vaadin-tab-horizontal-padding: 0.75rem;
   margin-right: auto;
   margin-left: auto;
 }
+
+// show horizontal scrollbar
+// needs to refactor, so utilize custom css properties
+:host([orientation="horizontal"][theme~="cxl-jobs-horizontal-slider"]) [part="tabs"] {
```

Comment:
`theme=cxl-tabs-slider` would probably make more sense for clarity, and it's not Jobs-only.